tests/test_data_loader.py

```diff
@@ -364,6 +380,14 @@ def test_iterable_dataset_shard(self):
         self.check_iterable_dataset_shards(dataset, seed, batch_size=4, drop_last=False, split_batches=True)
         self.check_iterable_dataset_shards(dataset, seed, batch_size=4, drop_last=True, split_batches=True)
 
+    def test_iterable_dataset_using_none_batch_size():
```

Comment:
This is part of a `unittest.TestCase`, so it needs `self`
```suggestion
    def test_iterable_dataset_using_none_batch_size(self):
```